src/jobs/test.yml

```diff
@@ -0,0 +1,33 @@
+description: |
+  Simple drop-in job to test your PHP application automatically.
+
+parameters:
+  version:
+    type: string
+    description: The `cimg/php` Docker image version tag.
+  setup:
+    type: steps
+    description: Provide any optional steps you would like to run prior to installing the node dependencies. This is a good place to install global packages.
+    default: []
+  test-command:
+    type: string
+    default: test
+    description: The name of the script within your composer.json which will run your tests.
+  app-dir:
+    type: string
+    default: ~/project
+    description: Path to the directory containing your composer.json file. Not needed if composer.json lives in the root.
+
+executor:
+  name: default
+  tag: << parameters.version >>
+
+steps:
+  - checkout
+  - steps: << parameters.setup >>
```

Comment:
Not 100% sure whether this works or not :thinking:

Reply:
This is stolen from https://github.com/CircleCI-Public/node-orb/blob/30ef903487c6db9bfe447dcde09589daa38f40c9/src/jobs/test.yml#L35

It works on my node builds.
Or, is there any insider knowledges?

Reply:
Ah, didn't notice that :)